src/components/Flows/FlowLogs.tsx

```diff
@@ -152,6 +156,40 @@ const LogsContainer = ({
     setAction(newAction);
   };
 
+  const fetchLogs = async () => {
+    setLogsLoaded(false);
+    (async () => {
+      try {
+        const data = await httpGet(
+          session,
+          `prefect/flow_runs/${flowRunId}/logs?task_run_id=${run.id}&offset=${Math.max(
+            flowRunOffset,
```

Comment:
@huzaifmalik786 in your `run` object, you will see that there is a field called `kind`. This can be `task-run` or `flow-run`. Your flow works correctly if its a `task-run`. 
For `flow-run` (airbyte sync job) kind, we need to use the `id` as the `flow_run_id` in logs api to fetch its logs. 

This was already happening , somehow we have changed it and its not working now.